docs/book/src/developer/tilt.md

```diff
@@ -55,7 +55,8 @@ podman machine start
 
 ## Create a kind cluster
 
-First, make sure you have a kind cluster and that your `KUBECONFIG` is set up correctly:
+First, make sure you have a kind cluster and that your `KUBECONFIG` is set up correctly.
+> **Note:** Execute the following in the current repository path and not from your local copy of `cluster-api` repository.
```

Comment:
Instead of this can we negate the statement and mention to execute from cluster-api-provider-ibmcloud repository?

Reply:
Changed it, PTAL.